functions.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+* These are all the functions that will be used into the Bot's code.
+*
+* No libraries are used in this project.
+*
+* @author		Giorgio Pais
+* @author		Giulio Coa
+* @author		Simone Cosimo
+* @author		Luca Zaccaria
+* @author		Alessio Bincoletto
+* @author		Marco Smorti
+*
+* @copyright	2020- Giorgio Pais <info@politoinginf.it>
+*
+* @license		https://choosealicense.com/licenses/lgpl-3.0/
+*/
+
+/**
+* Execute an HTTP(S) request.
+*
+* @param string $urlt The HTTP(S) endpoint.
+*
+* @return mixed The result of the request.
+*/
+function request(string $url) {
+	/**
+	* Replace the special character into the URL
+	*
+	* str_replace() Replace the first array of character with the second array
+	*/
+	$url = str_replace([
```

Comment:
Please use `urlencode` instead of str_replace.

Reply:
Alredy done